setup.cfg

```diff
@@ -0,0 +1,114 @@
+[metadata]
```

Comment:
Yep, this became possible with PEP 517 since the latest available `setuptools` get auto-provisioned according to `pyproject.toml` and then the hooks are called directly. Plus a sufficient version of `setuptools` emulates a dummy `setup.py` in-memory (it basically `eval()`s `import setuptools; setuptools.setup()` which picks up `setup.cfg` as per usual).